src/shogun/converter/EmbeddingConverter.cpp

```diff
@@ -18,7 +18,9 @@ CEmbeddingConverter::CEmbeddingConverter()
 {
 	m_target_dim = 1;
 	m_distance = new CEuclideanDistance();
+	SG_REF(m_distance);
```

Comment:
@karlnapf @lisitsyn check this commit and lemme know if you have objections, as then i'll just cherry-pick this one commit into develop....